src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,127 @@
+                String country = resultSet.getString("country");
+                Manufacturer manufacturer = new Manufacturer(name, country);
+                manufacturer.setId(id);
+                resultManufacturer = Optional.of(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't get manufacturer by id", e);
+        }
+        return resultManufacturer;
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> allManufacturers = new ArrayList<>();
+        String getAllManufacturers = "SELECT * FROM manufacturers "
+                + "WHERE is_deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getAllManufacturersStatement
+                        = connection.prepareStatement(getAllManufacturers)) {
+            ResultSet resultSet = getAllManufacturersStatement.executeQuery();
+            while (resultSet.next()) {
+                String name = resultSet.getString("name");
+                String country = resultSet.getString("country");
+                Long id = resultSet.getLong("id");
+                Manufacturer manufacturer = new Manufacturer(name, country);
+                manufacturer.setId(id);
+                allManufacturers.add(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't get all manufacturers from DB", e);
+        }
+        return allManufacturers;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateManufacturerRequest =
+                "UPDATE manufacturers SET name = ?"
+                        + ", country = ? WHERE id = ? AND is_deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateManufacturerStatement
+                        = connection.prepareStatement(
+                            updateManufacturerRequest,Statement.RETURN_GENERATED_KEYS)) {
+            updateManufacturerStatement.setString(1, manufacturer.getName());
+            updateManufacturerStatement.setString(2, manufacturer.getCountry());
+            updateManufacturerStatement.setLong(3, manufacturer.getId());
+            updateManufacturerStatement.executeUpdate();
+            ResultSet resultSet =
+                    updateManufacturerStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                Long id = resultSet.getObject(1, Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't get manufacturer by id", e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteManufacturerRequest =
+                "UPDATE manufacturers SET is_deleted = ?"
```

Comment:
```suggestion
                "UPDATE manufacturers SET is_deleted = TRUE"
```
you set it to true always anyway